stacker/lookups/output.py

```diff
@@ -0,0 +1,35 @@
+from collections import namedtuple
```

Comment:
lookups and handlers are essentially the same. any new lookup should have its own file within the `lookups` directory